javaparser-core/src/main/java/com/github/javaparser/ParserConfiguration.java

```diff
@@ -42,19 +42,36 @@
  */
 public class ParserConfiguration {
     public enum LanguageLevel {
+        /** Does no post processing or validation. Only for people wanting the fastest parsing. */
         RAW(null, null),
+        /** Always points to the most popular Java version. */
+        POPULAR(new Java8Validator(), null),
```

Comment:
CURRENT?

Reply:
You mean: add CURRENT? Or "this should be CURRENT"?

Reply:
I mean this should be current

Reply:
Java 9 is current, but Java 8 is popular. I thought it would be useful to have a "most commonly used" option. I can add current?

Reply:
CURRENT added :-)